drivers/wifi/esp_at/esp_offload.c

```diff
@@ -27,9 +27,13 @@ static int esp_listen(struct net_context *context, int backlog)
 
 static int _sock_connect(struct esp_data *dev, struct esp_socket *sock)
 {
-	char connect_msg[sizeof("AT+CIPSTART=000,\"TCP\",\"\",65535,7200") +
-			 NET_IPV4_ADDR_LEN];
-	char addr_str[NET_IPV4_ADDR_LEN];
+	char connect_msg[MAX(sizeof("AT+CIPSTART=000,\"TCP\",\"\",65535,7200") +
```

Comment:
Why do we need to use MAX() here? Isn't the latter parameter always larger?

Reply:
Second parameter is larger, indeed. I just tried to make it explicit how we calculate length. I can leave just the second statement and get rid of MAX() if you prefer that.

Reply:
No need to remove it if you did it on purpose, but perhaps a comment would help here to explain why the code looks like that.